pkg/cloud/metadata_k8s.go

```diff
@@ -60,21 +57,20 @@ func KubernetesAPIInstanceInfo(clientset kubernetes.Interface) (*Metadata, error
 		return nil, fmt.Errorf("error getting Node %s: %v", nodeName, err)
 	}
 
-	// Get node labels
-	labels := node.GetLabels()
-	keysList := []string{CloudInstanceIDLabel, PvmInstanceIdLabel}
 	instanceInfo := Metadata{}
-	for _, key := range keysList {
-		if val, ok := labels[key]; ok {
-			switch key {
-			case CloudInstanceIDLabel:
-				instanceInfo.cloudInstanceId = val
-			case PvmInstanceIdLabel:
-				instanceInfo.pvmInstanceId = val
-			}
-		} else {
-			return nil, fmt.Errorf("error getting label %s for node Node %s", key, nodeName)
+
+	// ProviderID format: ibmpowervs://<region>/<zone>/<service_instance_id>/<powervs_machine_id>
+	if node.Spec.ProviderID != "" {
+		providerId := node.Spec.ProviderID
+		klog.Infof("PROVIDER-ID: %s", providerId)
```

Comment:
I'm not if we are printing codename anywhere before this? otherwise this prints just an providerid and it becomes very difficult understand when we large number of nodes in the cluster.

I prefer printing node name also with this.

Reply:
Node name printed.